web/src/actions/apd.js

```diff
@@ -22,7 +22,6 @@ export const SAVE_APD_REQUEST = 'SAVE_APD_REQUEST';
 export const SAVE_APD_SUCCESS = 'SAVE_APD_SUCCESS';
 export const SAVE_APD_FAILURE = 'SAVE_APD_FAILURE';
 export const SELECT_APD = 'SELECT_APD';
-export const SET_KEY_PERSON_PRIMARY = 'SET_KEY_PERSON_PRIMARY';
```

Comment:
Since the first key person is now always primary and that one can't be removed, no need to set which one is primary.

![wipe it away](https://media.giphy.com/media/26u4lOMA8JKSnL9Uk/giphy.gif)